<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemFrame</class>
 <widget class="QWidget" name="ItemFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);</string>
    </property>
    <attribute name="title">
     <string>项目信息</string>
    </attribute>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>项目名称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>项目全称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>波长</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>主</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>副</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>100</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>方法周期</string>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>一点终点法</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>46</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>两点终点法</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>动力学法</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>260</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>小数位数</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_13">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>260</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>290</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>单位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_14">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>290</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>mg/l</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ug/l</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>g/l</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>330</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>检测方法</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_14">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>330</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>390</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>限值</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_15">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>390</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>390</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>低</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_15">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>390</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>k</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>370</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>判读</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_16">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>420</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>420</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>高</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_16">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>420</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>130</y>
       <width>211</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>两点终点法参数</string>
     </property>
     <widget class="QComboBox" name="comboBox_17">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>测量时机</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>130</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>动力学法参数</string>
     </property>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>测量间隔</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_17">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>测量次数</string>
      </property>
     </widget>
    </widget>
    <zorder>label_18</zorder>
    <zorder>lineEdit_9</zorder>
    <zorder>lineEdit_11</zorder>
    <zorder>label_25</zorder>
    <zorder>label_26</zorder>
    <zorder>label_27</zorder>
    <zorder>label_28</zorder>
    <zorder>lineEdit_12</zorder>
    <zorder>lineEdit_13</zorder>
    <zorder>label_29</zorder>
    <zorder>comboBox_13</zorder>
    <zorder>label_30</zorder>
    <zorder>comboBox_14</zorder>
    <zorder>label_31</zorder>
    <zorder>lineEdit_14</zorder>
    <zorder>label_32</zorder>
    <zorder>lineEdit_15</zorder>
    <zorder>label_33</zorder>
    <zorder>comboBox_15</zorder>
    <zorder>label_34</zorder>
    <zorder>comboBox_16</zorder>
    <zorder>label_35</zorder>
    <zorder>lineEdit_16</zorder>
    <zorder>groupBox_10</zorder>
    <zorder>groupBox_8</zorder>
    <zorder>groupBox_9</zorder>
   </widget>
   <widget class="QWidget" name="tab_5">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);</string>
    </property>
    <attribute name="title">
     <string>项目流程</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>621</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>方法流程</string>
     </property>
     <widget class="QLabel" name="label_61">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>step1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_62">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>step2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_63">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>step3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_64">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>step4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_65">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>step5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_66">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>溶液</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_67">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>体积(ul)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_68">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>孵育时间</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_25">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_26">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_27">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_28">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_41">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_29">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>473</width>
        <height>76</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBox_35"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_31"/>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBox_33"/>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="comboBox_34"/>
       </item>
       <item row="0" column="4">
        <widget class="QComboBox" name="comboBox_32"/>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_32"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_35"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_37"/>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEdit_39"/>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="lineEdit_42"/>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="lineEdit_33"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_34"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_36"/>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEdit_38"/>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="lineEdit_40"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>5</y>
       <width>611</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_60">
      <property name="text">
       <string>项目名称</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_31"/>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>真实样品空白</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_69">
      <property name="text">
       <string>替代试剂</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_30"/>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);</string>
    </property>
    <attribute name="title">
     <string>稀释重做</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget1">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>20</y>
     <width>241</width>
     <height>521</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>测试项目名称</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>538</y>
     <width>1221</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(149, 149, 149);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>新建</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>0</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
